examples/playbook_with_double_ext.ansible.yml

```diff
@@ -0,0 +1,5 @@
+- hosts: all
```

Comment:
I you use `playbook` keyword in filename, it will still use the old glob pattern. Maybe it would better to use something that is not already included in the glob patterns like `play_` or `pb_`. It would make much easier to test the change in behavior, as we would know that these files are not expected to be picked as ansible by default.

Reply:
Makes sense! Thanks.

Reply:
updated.